src/views/PeopleSearch/index.js

```diff
@@ -1013,8 +1027,8 @@ class PeopleSearch extends Component {
                           this.setState(
                             {
                               searchValues: {
-                                includeStudent: false,
-                                includeStaff: false,
+                                includeStudent: true,
+                                includeStaff: true,
                                 includeAlumni: false,
```

Comment:
Setting these in the `Clear All` button might have weird consequences for Alumni who are searching. Maybe use the personType state to determine what the "reset" values are?
```suggestion
                                includeStudent: this.state.personType.includes('alum') ? false : true,
                                includeStaff: true,
                                includeAlumni: this.state.personType.includes('alum') ? true : false,
```

Reply:
That makes sense. Thanks!